src/OrleansGCPUtils/Providers/Streams/PubSub/IPubSubDataAdapter.cs

```diff
@@ -7,12 +7,12 @@
 using System.Collections.Generic;
 using System.Linq;
 
-namespace Orleans.Providers.Streams
+namespace Orleans.Providers.GCP.Streams.PubSub
 {
     /// <summary>
     /// Converts event data to and from cloud queue message
     /// </summary>
-    public interface IGooglePubSubDataAdapter
```

Comment:
The current namespace is `Orleans.Providers.GCP.Streams.PubSub`. I think it dedup the name and its clear enough for users.